### DISCUSSION TITLE
dotnet new console

### DISCUSSION THREAD

**Discussion by eoliverdev** [balta-io/forum#294] [Tira Dúvidas] [answered]
Olá Balta, quando eu dou um **_'dotnet new console'_** no meu terminal o Program.cs não vem com toda estrutura pronta com namespace, class, etc... só vem 'Console.WriteLine("Hello, World!");'.

Já instalei as extensões e fui até novamente em https://balta.io/blog/visual-studio-code-instalacao-customizacao pra ver se não tinha alguma extensão.

Como faço pra já vir estruturado?

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @eoliverdev como vai?
Este vídeo aqui vai te mostrar as mudanças e como estruturar:
https://www.youtube.com/watch?v=PNasRBbo00g

**Comment by williamdev93** [comment]
Para vir estruturado:
dotnet new console -o nomeDoProjeto --use-program-main
